### PR TITLE
Add JSON error handler

### LINKED ISSUE #31: Add error handler
... so that always JSON is returned.

### PR DESCRIPTION
**Pull request by tsauerwein**
... so that always JSON is returned.

Closes #31

### CODE REVIEW

File: c2corg_api/views/__init__.py [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
 import datetime
 from colander import null
+from pyramid.httpexceptions import HTTPError, HTTPNotFound
+from pyramid.view import view_config
+from cornice import Errors
+from cornice.util import json_error
+
+
+@view_config(context=HTTPNotFound)
+@view_config(context=HTTPError)
```
Comment (asaunier) [inline c2corg_api/views/__init__.py]: Does this means that when an HTTPNotFound or HTTPError is detected, http_error_handler() will automatically be triggered to add the JSON error output as the response?

Comment (tsauerwein) [inline c2corg_api/views/__init__.py]: Yes.

Actually HTTPNotFound [should](http://docs.pylonsproject.org/projects/pyramid/en/latest/api/httpexceptions.html#pyramid.httpexceptions.HTTPNotFound) also be an instance of HTTPError, but I had to specify it explicitly.

Comment (asaunier) [inline c2corg_api/views/__init__.py]: OK, looks good to me.

Out of curiosity: the error names (such as "Conflict" - see the tests above) are set by Cornice?

Comment (tsauerwein) [inline c2corg_api/views/__init__.py]: By Pyramid in this case. When you raise/return HTTPConflict, Pyramid sets the title (see [here](http://docs.pylonsproject.org/projects/pyramid/en/latest/_modules/pyramid/httpexceptions.html#HTTPConflict)).

**Comment by tsauerwein** [comment]
Thanks for the review, Alex.

**Pull request merged**